website/docs/getting-started/setup/installation-guides/google-cloud-run/setup-to-integrate-sso.mdx

```diff
@@ -0,0 +1,127 @@
+
+3. On the **Choose your database engine** panel of the **Create an instance** page, click **Choose PostgreSQL**.
+
+4. In the **Instance ID** field of the **Instance info** pane, enter an ID for your instance. Eg: `postgres`.
+
+5. In the **Password** field, enter a password for the postgres user.
+
+6. Select **PostgreSQL 12** in the **Database version** dropdown list.
+
+7. Select **Enterprise** in the **Choose a Cloud SQL edition** panel.
+
+8. In the **Choose region and zonal availability** section:
+    * In the **Region** dropdown list, select the region for your instance.   
+    * Select **Single zone** under **Zonal availability**.
+
+9. In the **Customize your instance** section, click **SHOW CONFIGURATION OPTIONS** to display the groups of settings.
+
+10. Under the **Storage** section: 
+    * Select **HDD** under **Storage type**.
+    * Select **10 GB** under **Storage capacity**.
+
+11. Under the **Connections** section, select the **Public IP** checkbox.
+
+12. Under the **Data protection** section, deselect the **Automate daily backups** and **Enable deletion protection** checkboxes.
+
+13. Under the **Maintenance** section:
+    * Select **Any window** in the **Maintenance window** dropdown list. 
+    * Select **Any** in the **Order of update** dropdown list. 
+
+14. Click **Create Instance**.
+
+### Create database
+
+1. In the Google Cloud console, go to the [Cloud SQL Instances](https://console.cloud.google.com/sql) page.
+
+2. To open the Overview page of an instance, click the instance name.
+
+3. Select **Databases** from the SQL navigation menu.
+
+4. Click **Create database**.
+
+5. In the **New database** dialog, specify the name of the database.
+
+6. Click **Create**.
+
+### Create users
+
+1. In the Google Cloud console, go to the [Cloud SQL Instances](https://console.cloud.google.com/sql) page.
+
+2. To open the Overview page of an instance, click the instance name.
+
+3. Select **Users** from the SQL navigation menu.
+
+4. Click **Add user account**.
+
+5. In the **Add a user account to instance instance_name** page, select **Built-in authentication** (the default) and add a **User name** and **Password**.
+
+6. Click **Add**.
+
+## Change Appsmith installation
+
+* Edit the Appsmith instance, and go to the **Autoscaling** section.
+* Under **Environment variables**, click **Add Variable** to add below variable in the **Name** and **Value** text boxes as shown below: <br/>
+
+| Name | Value |
+|------------------------|------------------------------------------| 
+|**APPSMITH_KEYCLOAK_DB_URL** | `jdbc:postgresql:///<DATABASE_NAME>?cloudSqlInstance=<INSTANCE_CONNECTION_NAME>&socketFactory=com.google.cloud.sql.postgres.SocketFactory&user=<POSTGRESQL_USER_NAME>&password=<POSTGRESQL_USER_PASSWORD>`|
+| **APPSMITH_KEYCLOAK_DB_USERNAME** |`appsmith`|
+| **APPSMITH_KEYCLOAK_DB_PASSWORD** |`appsmith`|
+| **APPSMITH_KEYCLOAK_DB_DRIVER** |`postgresql`|
+* --- Add steps to update the installation ---
```

Comment:
@pratapaprasanna - Here the steps are needed for updating or redeploying the installation.

Reply:
Added the steps after our discussion.